### PR TITLE
Product list bulk delete and duplicate actions

### PR DESCRIPTION
**Pull request by zuk3975**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add missing bulk actions: delete, duplicate in new product list.Screenshot from 2022-05-24 15-32-27
| Type?             | refacto
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| Related PRs       | ~
| How to test?      | Go to experimental product list, check that bulk delete and bulk duplicate actions work. To use bulk action you need to check checkboxes of products which should be affected, and use bulk action dropdown at the top of the list to select the action. Bulk deletion should delete selected products, bulk duplication should duplicate selected products (products with same name prefixed with "copy of .." will appear
| Possible impacts? | ~

### CODE REVIEW

File: src/Core/Grid/Definition/Factory/ProductGridDefinitionFactory.php [hunk 1]
```diff
@@ -409,7 +409,7 @@ protected function getBulkActions()
                 $this->trans('Activate selection', [], 'Admin.Actions'),
                 $this->trans('Activating %total% products', [], 'Admin.Actions'),
                 $this->trans('Activating %done% / %total% products', [], 'Admin.Actions'),
-                'radio_button_checked',
+                'radio_button_unchecked',
```
Comment (jolelievre) [inline src/Core/Grid/Definition/Factory/ProductGridDefinitionFactory.php]:
```suggestion
                'radio_button_checked',
```

File: src/Core/Grid/Definition/Factory/ProductGridDefinitionFactory.php [hunk 2]
```diff
@@ -421,6 +421,22 @@ protected function getBulkActions()
...
+                $this->trans('Deleting %done% / %total% products', [], 'Admin.Actions'),
+                'radio_button_unchecked'
+            ))
+            ->add($this->buildAjaxBulkAction(
+                'bulk_duplicate_ajax',
+                'admin_products_v2_bulk_duplicate',
+                $this->trans('Duplicate selection', [], 'Admin.Actions'),
+                $this->trans('Duplicating %total% products', [], 'Admin.Actions'),
+                $this->trans('Duplicating %done% / %total% products', [], 'Admin.Actions'),
+                'radio_button_unchecked'
```
Comment (jolelievre) [inline src/Core/Grid/Definition/Factory/ProductGridDefinitionFactory.php]:
```suggestion
                'content_copy'
```

File: src/Core/Grid/Definition/Factory/ProductGridDefinitionFactory.php [hunk 3]
```diff
@@ -421,6 +421,22 @@ protected function getBulkActions()
                 'radio_button_unchecked',
                 ['productStatus' => false]
             ))
+            ->add($this->buildAjaxBulkAction(
+                'bulk_delete_ajax',
+                'admin_products_v2_bulk_delete',
+                $this->trans('Delete selection', [], 'Admin.Actions'),
+                $this->trans('Deleting %total% products', [], 'Admin.Actions'),
+                $this->trans('Deleting %done% / %total% products', [], 'Admin.Actions'),
+                'radio_button_unchecked'
```
Comment (jolelievre) [inline src/Core/Grid/Definition/Factory/ProductGridDefinitionFactory.php]:
```suggestion
                'delete'
```

**Review by jolelievre — CHANGES_REQUESTED** [review]

**Review by jolelievre — APPROVED** [review]
Thanks @zuk3975

**Review by matks — APPROVED** [review]

**Pull request merged**